models/migrations/v1_21/v276.go

```diff
@@ -0,0 +1,18 @@
+// Copyright 2023 The Gitea Authors. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+package v1_21 //nolint
+
+import (
+	actions_model "code.gitea.io/gitea/models/actions"
+
+	"xorm.io/xorm"
+)
+
+func AddPermissions(x *xorm.Engine) error {
+	type ActionRunJob struct {
+		Permissions actions_model.Permissions `xorm:"JSON TEXT"`
```

Comment:
We need to copy the Permissions struct here because in future that struct maybe changed.

Reply:
Excellent point. I'll make that change.

Reply:
Wait, does it even matter, since it's stored as JSON? Should I just put `interface{}` instead of `actions_model.Permissions`?